android-automation/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:3.8
```

Comment:
I'd suggest using `python:3.8-alpine` here instead, as it will run on the latest `alpine` release (currently 3.11), and not with this one which was released more than a year ago.

Moreover it won't be needed to install python anymore, as it's already there 🎉 

```suggestion
FROM python:3.8-alpine
```

Reply:
Done, as we discussed via Slack